### ISSUE TITLE
Exception has been thrown by the target of an invocation.

### ISSUE THREAD

**Issue by ySimbaa** [DurtyFree/durty-cloth-tool#356]
I have the following error:
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.IO.DirectoryNotFoundException: Could not find a part of the path 'E:\StateV\Neuer Ordner\rest02.dctproj'.
   at Microsoft.Win32.SafeHandles.SafeFileHandle.CreateFile(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options)
   at Microsoft.Win32.SafeHandles.SafeFileHandle.Open(String fullPath, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.File.OpenHandle(String path, FileMode mode, FileAccess access, FileShare share, FileOptions options, Int64 preallocationSize)
   at System.IO.File.WriteAllBytes(String path, Byte[] bytes)
   at CCCp3OsSfTb1llhIuQkJ.c9ysrR0Ya75(String , Byte[] , CCCp3OsSfTb1llhIuQkJ )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.bi4B1OPL0R(Object  , Object  )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.KwFlH8LdEgEH0ue2oAPA(Object , Object )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.sHGBKPjC4o(String  , Boolean  , Boolean  )
   at LPEqgNBytRhe3ZetZte.svriccBYH74MFTT8qG4.nbCBbYOR6Y()
   at rQmql1uCtDYM7E8xIY4.gFe59quMjtCH6qNFgyr.BkvuiMlUtN(Object  , RoutedEventArgs  )
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Span`1& arguments, Signature sig, Boolean constructor, Boolean wrapExceptions)
   at System.Reflection.RuntimeMethodInfo.Invoke(Object obj, BindingFlags invokeAttr, Binder binder, Object[] parameters, CultureInfo culture)
   at System.Delegate.DynamicInvokeImpl(Object[] args)
   at System.Windows.RoutedEventArgs.InvokeEventHandler(Delegate genericHandler, Object genericTarget)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at Telerik.Windows.Controls.RadMenuItem.RaiseClickEvent()
   at Telerik.Windows.Controls.RadMenuItem.OnClickImpl()
   at Telerik.Windows.Controls.RadMenuItem.OnClick()
   at Telerik.Windows.Controls.RadMenuItem.ClickItem()
   at Telerik.Windows.Controls.RadMenuItem.HandleMouseUp()
   at Telerik.Windows.Controls.RadMenuItem.OnMouseLeftButtonUp(MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

App Version: **3.0.0.8**
Project targets: **AltV**
DLC name: **femalerest_02**
Collection name: **femalerest_02 // **
Project path: **E:\StateV\Neuer Ordner\clothproject_femalerest02**
Output path: **E:\StateV\Neuer Ordner**
Disable Extra Ymt Creation: **False**
Create Subfolder For Cloth Drawable Types: **False**
Replace settings enabled: **False // DLC: NoDlc**
Save File Version: **5**
```

Cloth files:
`[Please provide cloth files (ydd, ytd), cloth project and any other related files here + describe reproduction steps]`

**Comment by DurtyFree** [comment]
Should be handled next update

**Comment by DurtyFree** [comment]
Fixed in v3.1 https://github.com/DurtyFree/durty-cloth-tool/releases/tag/3.1.0.0
